File: src/test/java/preprocessor/Preprocess.java
package preprocessor;

import com.kingmang.lazurite.parser.preprocessor.Preprocessor;
import org.junit.Test;

import static org.junit.Assert.assertEquals;


public class Preprocess {

    @Test
    public void testJIncludePreprocess() {
        String code = "#jInclude \"java.util.Stack\"";
        String input = Preprocessor.preprocess(code);
        String output = "using \"lzr.reflection\"; Stack = JClass(\"java.util.Stack\")\n";
        assertEquals(input, output);
    }
}

File: src/test/java/preprocessor/Preprocess.kt
package preprocessor

import com.kingmang.lazurite.parser.preprocessor.Preprocessor.preprocess
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class Preprocess {
    @Test
    fun testJIncludePreprocess() {
        val code = "#jInclude \"java.util.Stack\""
        val input = preprocess(code)
        val output = "using \"lzr.lang.reflection\"; Stack = JClass(\"java.util.Stack\")\n"
        Assertions.assertEquals(input, output)
    }
}
